src/components/CProductItemCard/index.tsx

```diff
@@ -2,9 +2,11 @@ import { StaticImageData } from 'next/image';
 
 import CItemCard from '../CItemCard';
 import CMethods, { MethodType } from '../CMethods';
+import CTokenLabel from '../CTokenLabel';
 
 export type TokensType = {
-  symbol: string;
```

Comment:
this causes conflict with new PR

Reply:
I can't change it because of the select component